test/common/object-diff.ts

```diff
@@ -0,0 +1,297 @@
+import {objectsAreSame, policiesAreSame} from "../../src/services/object-diff";
+
+jest.setTimeout(30000);
+
+describe("policiesAreSame", () => {
+    describe("when cloud and DB policies are exactly the same", () => {
```

Comment:
super nit - I think all other files have a tab space of 2 and this one has 4 spaces. it would be cool to enforce that in the style, but all super minor